setup.py

```diff
@@ -45,7 +45,9 @@ def get_version(rel_file='plugin.yaml'):
     install_requires=[
         "cloudify-common>=4.5.5",
         "cloudify-utilities-plugins-sdk==0.0.28",  # obfuscate_passwords
-        "pyyaml==3.12"
+        "pyyaml==3.12",
+        'oauth2client'  # used in GCPServiceAccountAuthentication
+
     ],
     test_requires=[
         "cloudify-common>=4.5.5",
```

Comment:
IMO, we don't need to support anything before 5.0.5. But ask Ofer.

Reply:
Asked. Its good like that for now.